client/src/Backoffice/components/Main/components/modals/AddRide/NewRideInfo/NewRideInfo.tsx

```diff
@@ -77,6 +86,48 @@ function NewRideInfo() {
             </FormHelperText>
           )}
         </FormControl>
+
+        <FormControl>
+          <DatePicker
+            label="תאריך איסוף מבוקש"
+            defaultValue={dayjs()}
+            maxDate={dayjs().add(3, 'day')}
+            disablePast
+            onChange={(date) => {
+              setValue('pickupDateTime', date ? date.toDate() : undefined);
+            }}
+            format="YYYY-MM-DD"
+            slots={{
+              textField: DayTextField
+            }}
+          />
+        </FormControl>
+        <FormControl sx={{ width: '100%' }} required>
+          <InputLabel id="demo-multiple-name-label" required>
+            כמה זמן רלוונטי
+          </InputLabel>
+          <Select
```

Comment:
@aviv-taori I think the value of this select should be the number of hours, not a string with the word `שעות`